egui/src/containers/scroll_area.rs

```diff
@@ -627,14 +627,14 @@ impl Prepared {
                 }
             } else {
                 let stop_speed = 20.0; // Pixels per second.
-                let friction_coeff = 1000.0; // Pixels per second squared.
+                let friction_coeff = 68.0;
                 let dt = ui.input().unstable_dt;
 
                 let friction = friction_coeff * dt;
                 if friction > state.vel.length() || state.vel.length() < stop_speed {
                     state.vel = Vec2::ZERO;
                 } else {
-                    state.vel -= friction * state.vel.normalized();
+                    state.vel = state.vel / friction;
```

Comment:
With a 120Hz display, `dt = 1/120`, so `friction = 68.0 / 120 = 0.5666…`, which means each frame the velocity will increase, which is a very serious bug to introduce.

Reply:
The idea of doing `x * dt` is no maintain same travel vs various frame rates. if it behaves opposite, then we should divide instead of multiplicate. I'm agree with that I will test it.